grpc/service_config/service_config.proto

```diff
@@ -190,6 +190,31 @@ message PickFirstConfig {}
 // Configuration for round_robin LB policy.
 message RoundRobinConfig {}
 
+// Configuration for priority LB policy.
+message PriorityLoadBalancingPolicyConfig {
+  // A map of name to child policy configuration.
+  // The names are used to allow the priority policy to update
+  // existing child policies instead of creating new ones every
+  // time it receives a config update.
+  message Child {
+    repeated LoadBalancingConfig config = 1;
+  }
+  map<string, Child> children = 1;
+
+  // A list of child names in decreasing priority order
+  // (i.e., first element is the highest priority).
+  repeated string priorities = 2;
```

Comment:
Observation: you could eliminate the map to simplify construction and usage of this message:

```proto
message Child {
  string name = 1;
  repeated LoadBalancingConfig config = 2;
}
repeated Child priorities = 1;
```

Reply:
A map seems like a more natural structure here, since that clearly indicates that there should be only one value for any given name.  It also has a much more natural JSON representation.